repository-data/webfiles/src/main/resources/site/freemarker/hee/pages/homepage-main.ftl

```diff
@@ -14,87 +14,84 @@
 <#else>
     <#assign heroType=' class="nhsuk-hero__wrapper"'>
 </#if>
-<section class="nhsuk-hero${heroImgClass}"${backgroundImage}>
-    <div class="nhsuk-width-container">
-        <div class="nhsuk-grid-row">
-            <div class="nhsuk-grid-column-two-thirds">
-                <div${heroType}>
-                    <h1 class="nhsuk-u-margin-bottom-3">${document.title}</h1>
-                    <#if document.summary??>
-                        <p class="nhsuk-body-l nhsuk-u-margin-bottom-0"><@hst.html formattedText="${document.summary!?replace('\n', '<br>')}"/></p>
-                    </#if>
-                    <span class="nhsuk-hero__arrow" aria-hidden="true"></span>
+<main class="page" id="maincontent" role="main">
+    <div class="page__header">
+        <section class="nhsuk-hero${heroImgClass}"${backgroundImage}>
+            <div class="nhsuk-width-container">
```

Comment:
I think this needs to be wrapped with `<div class="nhsuk-hero__overlay">...</div>` per Hero component FE spec.